picard/util/tags.py

```diff
@@ -92,6 +92,15 @@
     'musicbrainz_originalartistid': N_('MusicBrainz Original Artist Id'),
     'originalalbum': N_('Original Album'),
     'musicbrainz_originalalbumid': N_('MusicBrainz Original Release Id'),
+    'movementname': N_('Movement Name (iTunes Classical)'),
+    'musicbrainz_movementid': N_('MusicBrainz Movement Work Id'),
+    'movementnumber': N_('Movement Number (iTunes Classical)'),
+    'movementtotal': N_('Movement Total (iTunes Classical)'),
+    'itunesgrouping': N_('Grouping (iTunes Classical)'), # ID3 GRP1 introduced in iTunes 12.5.4.42 Dec 2016
+    'ituneswork': N_('Work (iTunes Classical)'), # mp4 ©wrk introduced in iTunes 2016
+    '~movementshow': N_('Movement Show (iTunes Classical)'),
```

Comment:
If this is written to files, remove the `~`.

Reply:
It is written to MP4 but not ID3 which is why I used a `~`. But on balance I agree it should be without it.